### PR TITLE
Test akka-sample-persistence-dc-scala on travis

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: akka-sample-persistence-dc-scala/README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ which is available as a commercial feature as part of the [Lightbend Subscriptio
 
 ## How to run
 
+You will need to have the `protobuf` compiler installed to convert `proto` definitions to object code.
```
Comment (johanandren) [inline akka-sample-persistence-dc-scala/README.md]: Any specific version of protobuf?

File: .travis.yml [hunk 2]
```diff
@@ -44,6 +44,7 @@ env:
     - CMD="cd akka-sample-persistence-java && sbt test"
     - CMD="cd akka-sample-persistence-java && mvn test"
     - CMD="cd akka-sample-persistence-scala && sbt test"
+    - CMD="mkdir ~/.lightbend && echo $LB_CRED | base64 -d > ~/.lightbend/commercial.credentials && sudo apt-get update && sudo apt-get install protobuf-compiler && cd akka-sample-persistence-dc-scala && sbt test"
```
Comment (chbatey) [inline .travis.yml]: :+1:

**Comment by chbatey** [comment]
Failure was unrealted, the cqre failure

**Comment by chbatey** [comment]
Fixes #161 https://github.com/akka/akka-samples/issues/50

File: akka-sample-persistence-dc-scala/README.md (hunk 1 above)
Comment (raboof) [reply akka-sample-persistence-dc-scala/README.md]: Not that I know specifically.. 'the default' both on my laptop and on ubuntu was OK

**Pull request merged**